lumicks/pylake/tests/test_channels.py

```diff
@@ -106,6 +155,87 @@ def test_calibration_continuous_channels():
     assert nested_slice.calibration[0]["Calibration Data"] == 120
 
 
+def test_arithmetic_operations():
```

Comment:
There is a significant amount of repetition in this function.  
I'm not sure what's the best strategy, but I see a lot of it could be reused for different channel type and also between raw lists a _numpy_ arrays.

Reply:
Yes, there is a significant amount of repetition, but I figured it'd be easier to interpret a failing test when each line can be attributed to a specific underlying datatype being tested.

Anyways, I changed it to avoid the repetition now. It's true that the repetition is a bit meh. :)